ops/deployer/gauntlet/gauntlet.go

```diff
@@ -0,0 +1,534 @@
+	_, err := exec.LookPath("terrad")
+	if err != nil {
+		return GauntlerDeployer{}, errors.New("'terrad' is not installed")
+	}
+
+	// check if yarn is installed
+	yarn, err := exec.LookPath("yarn")
+	if err != nil {
+		return GauntlerDeployer{}, errors.New("'yarn' is not installed")
+	}
+	fmt.Printf("yarn is available at %s\n", yarn)
+
+	// Change path to root directory
+	cwd, _ := os.Getwd()
+	os.Chdir(filepath.Join(cwd, "../"))
+
+	fmt.Println("Installing dependencies")
+	if _, err = exec.Command(yarn).Output(); err != nil {
+		return GauntlerDeployer{}, errors.New("error install dependencies")
+	}
+
+	// Generate Gauntlet Binary
+	fmt.Println("Generating Gauntlet binary...")
+	_, err = exec.Command(yarn, "bundle").Output()
+	if err != nil {
+		return GauntlerDeployer{}, errors.New("error generating gauntlet binary")
+	}
+
+	os.Setenv("SKIP_PROMPTS", "true")
+
+	// Check gauntlet works
+	os.Chdir(cwd) // move back into ops folder
+	gauntletBin := filepath.Join(cwd, "../")
+	gauntlet, err := relayUtils.NewGauntlet(gauntletBin)
+
+	if err != nil {
+		return GauntlerDeployer{}, err
+	}
+
+	keyID := config.Require(ctx, "TERRA-DEPLOYER")
+	chainID := config.Require(ctx, "CL-RELAY_CHAINID")
+
+	return GauntlerDeployer{
+		gauntlet: gauntlet,
+		network:  "local",
+		Account:  make(map[int]string),
+		chainID:  chainID,
+		keyID:    keyID,
+		args:     []string{"--from", keyID, "--chain-id", chainID, "--gas=auto", "--gas-adjustment=1.25", "--fees=100000uluna", "--broadcast-mode=block", "-y", "-o=json"},
+	}, nil
+}
+
+func (t *GauntlerDeployer) Load() error {
+	msg := utils.LogStatus("Uploading contract artifacts")
+	err := t.gauntlet.ExecCommand(
+		"upload",
+		t.gauntlet.Flag("network", t.network),
+		"link",
+		"ocr2",
+		"access_controller",
+	)
+
+	if err != nil {
+		return errors.New("Uploading contracts failed")
+	}
+	return msg.Check(nil)
+}
+
+func (t *GauntlerDeployer) DeployLINK() error {
+	fmt.Println("Deploying LINK Token...")
+	err := t.gauntlet.ExecCommand(
+		"token:deploy",
+		t.gauntlet.Flag("network", t.network),
+	)
+	if err != nil {
+		return errors.New("LINK contract deployment failed")
+	}
+
+	report, err := t.gauntlet.ReadCommandReport()
+	fmt.Println(report)
+	if err != nil {
+		fmt.Println(err)
+		return errors.New("report not available")
+	}
+
+	linkAddress := report.Responses[0].Contract
+	t.Account[LINK] = linkAddress
+
+	msg := utils.LogStatus("Deployed LINK token")
+
+	return msg.Check(nil)
+}
+
+type OCRinit struct {
+	LinkToken                 string `json:"linkToken"`
+	MinAnswer                 string `json:"minAnswer"`
+	MaxAnswer                 string `json:"maxAnswer"`
+	BillingAccessController   string `json:"billingAccessController"`
+	RequesterAccessController string `json:"requesterAccessController"`
+	Decimals                  int    `json:"decimals"`
+	Description               string `json:"description"`
+}
+
+func (t *GauntlerDeployer) DeployOCR() error {
```

Comment:
@RodrigoAD could we use a flow command here, instead of invoking individual commands and essentially rebuilding the Gauntlet's flow command?

Reply:
We can, but I'd say to use individual commands as they give more control, and the flow command is not as maintained as the individual commands

Reply:
After discussing it, we should be able to use Flow Commands. Dev team uses them, and this should force us to keep them maintained.

Reply:
Not going to make this change right now, since it seems like the flow command is out of date